crawl-ref/source/dat/descript/status.txt

```diff
@@ -333,6 +333,36 @@ You are silenced. This makes reading scrolls, casting spells, invoking divine
 abilities, or shouting impossible. The oppressive, unnatural effect greatly
 hampers your stealth.
 %%%%
+Engulf status
+
+{{
+    desc = "You are engulfed in water"
+
+    if you.race() == "Octopode"
+       or you.race() == "Merfolk"
+       or you.race() == "Barachi"
+       or you.transform() == "hydra"
+       or you.transform() == "ice" then
+        desc = desc .. "."
+    elseif you.race() == "Ghoul"
+           or you.race() == "Mummy"
+           or you.race() == "Vampire"
+           or you.race() == "Gargoyle"
+           or you.race() == "Grey Draconian" then
+        desc = desc .. ". It will take longer to move and escape the water" ..
+                       " elemental engulfing you."
+    else
+        desc = desc .. " and are unable to breathe. You will take increasing" ..
```

Comment:
Using `[[` and `]]` for multiline string literals might be nicer

Reply:
That still requires concatenation with `.. [[ "text on line" ]]`, no?